src/main/scala/beam/utils/gtfs/GTFSToShape.scala

```diff
@@ -53,9 +53,7 @@ object GTFSToShape extends LazyLogging {
 
       try {
         val agencies = GTFSReader.readAgencies(zipFile, zipFileName)
-        val defaultAgency =
-          if (agencies.size == 1) Some(agencies.head)
-          else None
+        val defaultAgency = if (agencies.size == 1) agencies.headOption else None
```

Comment:
This seems like a bug - should the test be > 0? In which case it is just headOption

Reply:
In this case does not sounds like a bug to me. basically states that a `defaultAgency` will exists only in case there is only one agency. in case of 2 or more there is no default.